application/models/TemplateConfiguration.php

```diff
@@ -545,6 +545,7 @@ public function getDescription()
           // Note: if no twig statement in the description, twig will just render it as usual
         try {
             $sDescription = App()->twigRenderer->convertTwigToHtml($this->template->description);
+            $sDescription = (new LSYii_Validators())->xssFilter($sDescription);
```

Comment:
My opinion : the description must be filtered when show.

For super admin too.

Reply:
> My opinion : the description must be filtered when show.

As the filter is on the getDescription() method, somehow the description is also filtered when shown ...
![image](https://github.com/LimeSurvey/LimeSurvey/assets/4942495/36d12e7c-4a58-47d4-b026-730d97506981)

> For super admin too.

It is applued to all users.

Reply:
You're right, sorry. But : totally unclear usage of a validator …